v0/destinations/zendesk/transform.js

```diff
@@ -453,32 +477,20 @@ async function processIdentify(message, destinationConfig, headers) {
   );
   const url = endPoint + category.createOrUpdateUserEndpoint;
   const returnList = [];
+  let userId = await getUserId(message, headers);
```

Comment:
If `searchByExternalId` is enabled we don't need to use this. Because if they are creating the user then also this get call going to fail and if they are updating the email then also the get call going to fail. So it would be better to search with externalId i.e. userId only.

Reply:
updated